site/content/en/docs/formats/mars.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'MARS'
+linkTitle: 'MARS'
+description: ''
+weight: 14
+---
+
+## Format specification
+
+MARS is a dataset for and motion analysis and person identification task,
+and this dataset it's extension of Market-1501 dataset format.
+MARS dataset is available for downloading
+[here](http://zheng-lab.cecs.anu.edu.au/Project/project_mars.html)
+
+Supported types of annotations:
+- `Label`
+
+Required attributes:
+- `is_distractors` (bool): True when image with distractors,
+    which negatively affect retrieval accuracy
+- `is_junk`: True for junk image which do not affect retrieval accuracy;
+- `pedestrian_id`: four-digit number in format `%04d`;
+- `camera_id`: one-digit number;
+- `track_id`: four-digit number in format `%04d`;
+- `frame_id`: three-digit number in format `%03d`, that mean number of
+  frame within this track. For the tracks, their names are accumulated
+  for each ID, but for frames, they start from "0001" in each track.
+
+
+## Import MARS dataset
+
+Use these instructions to import MARS dataset into Datumaro project:
+
+```bash
+datum create
+datum add -f mars ./dataset
+```
+
+> Note: the directory with dataset should be subdirectory of the
+> project directory.
+
+```
+mars_dataset
+├── <bbox_subset_name1>
```

Comment:
Will this format support metafile?

Reply:
I'm not sure that this format needs a metafile.

Reply:
Probably, I can agree.